src/api/archive_unstable_storageDiffStop.md

```diff
@@ -0,0 +1,11 @@
+# archive_unstable_storageDiff
```

Comment:
misnamed title :)

But also; worth calling it just `archive_unstable_stop`?

Reply:
Made the subscription a method after all, thanks